api/tests/opentrons/protocol_engine/state/test_command_state.py

```diff
@@ -728,6 +760,7 @@ def test_recovery_target_tracking() -> None:
     # c3 failed, but not recoverably.
     assert subject_view.get_recovery_target() is None
     assert not subject_view.get_recovery_in_progress_for_command("c3")
+    assert subject_view.get_has_entered_recovery_mode() is True
```

Comment:
Nitpick:

The placement of this `assert` is confusing because it's grouped with a comment that says error recovery *does not* apply. The result of `get_has_entered_error_recovery_mode()` happens to be `True` here not because of the state updates right above, but because of state updates earlier in the test.

Let's put a blank line before it so it looks like it's its own group.

```suggestion

    assert subject_view.get_has_entered_recovery_mode() is True
```